src/main/scala/symsim/ValueFunction.scala

```diff
@@ -0,0 +1,61 @@
+package symsim
+
+import cats.Monad
+import cats.syntax.functor.*
+import cats.syntax.flatMap.*
+import cats.syntax.foldable.*
+import cats.syntax.option.*
+import org.scalacheck.Gen
+import org.typelevel.paiges.Doc
+import symsim.concrete.{ConcreteExactRL, Randomized}
+
+
+trait ValueFunction[State, ObservableState, Action, Reward, Scheduler[_]]:
+   type VF
```

Comment:
Make all the indentation steps in this file 2 spaces (not 3). For some time, wrongly, I was switching to 3-space indentation, but the standard in Scala 3 is two spaces, so for new files we should use the correct one (and for the old files we improve when we touch them, where we touch them, to keep commits smaller and easier to review).